build.gradle

```diff
@@ -149,7 +149,6 @@ configure(allProjs) {
                 "-language:implicitConversions", "-language:existentials", "-language:postfixOps"
         ]
     }
-    compileScala.scalaCompileOptions.additionalParameters += "-optimize"
```

Comment:
why remove optimization option?

Reply:
`-optimize` is deprecated. I've now added the new optimization flags that replace it.